menu/fail2ban_menu.sh

```diff
@@ -44,8 +45,32 @@ case "$choice" in
             msg_box "Fail2ban isn't installed. Please run 'sudo bash /var/scripts/menu.sh' to install it." "$SUBTITLE"
         fi
     ;;&
+    *"Fail2ban-UnbanIP"*)
+        SUBTITLE="Fail2ban Unban IP"
+        if is_this_installed fail2ban && [ -f "/etc/fail2ban/filter.d/nextcloud.conf" ]
+        then
+            UNBANIP="$(input_box_flow "Enter the IP address that you want to unban.")" "$SUBTITLE"
+            if ! iptables -L -n | grep -qv "$UNBANIP"
+            then
+                msg_box "It seems that $UNBANIP isn't banned. Please try again."
+                run_script MENU fail2ban_menu
```

Comment:
Running the script again instead of a loop to be able to exit out, instead of getting into a possibly endless loop.